price/priceService.ts

```diff
@@ -0,0 +1,21 @@
+const baseUrl:string = 'https://dydx-testnet.imperator.co/v4';
```

Comment:
This should be updated to mainnet url. In addition, please define this url as a constant in another file and export it, so we can reuse it in other places, too.